model/metabolites.tsv

```diff
@@ -4514,6 +4516,7 @@ mets	metsNoComp	metBiGGID	metKEGGID	metHMDBID	metChEBIID	metPubChemID	metLipidMa
 "MAM02642c"	"MAM02642"	"octa"	"C06423"	"HMDB0000482"	"CHEBI:28837"	"379"	"LMFA01010008"	""	""	"octa"	"MNXM750"	"m02642c"	"m02642c"
 "MAM02642e"	"MAM02642"	"octa"	"C06423"	"HMDB0000482"	"CHEBI:28837"	"379"	"LMFA01010008"	""	""	"octa"	"MNXM750"	"m02642s"	"m02642s"
 "MAM02643c"	"MAM02643"	""	"C05752"	""	""	""	""	""	"HC01595"	"HC01595"	"MNXM979"	"m02643c"	"m02643c"
+"MAM02643m"	"MAM02643"	""	"C05752"	""	""	""	""	""	""	""	"MNXM979"	""	""
```

Comment:
The annotation of new met `MAM02643m` could refer to `MAM02643c`. Missing here are the matching IDs from Recon3D and HepatoNET1.

Reply:
I intentionally didn't include those IDs since, as far as I know, those metabolites also only existed in the cytosolic compartments of those models and don't have mitochondrial equivalents. I can add them if it's not important to ensure that the compartments match

Reply:
Okay, if there are metabolites that don't match the same compartment, I agree not to add them.